CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.13.1
+### Fixed
+- Fix npm audit #6028
+- Fix npm audit #6001
+- fix: Avoid optional before required parameter #5820
+- fix: permission check for cloning board #5853
```

Comment:
Sorry just noticed there was https://github.com/nextcloud/deck/pull/6038 which was good to have merged still for this release, could you add that to the changelog and rebase this branch to be on top of the latest stable29 after the merge?